lib/tasks/evm_dbsync.rake

```diff
@@ -74,7 +74,7 @@ namespace :evm do
       unless (total = RrPendingChange.count) == 0
         require 'progressbar'
         require 'rubyrep'
-        pbar = ProgressBar.new("Backlog", total)
+        pbar = ProgressBar.create(:title => "Backlog", :total => total, :autofinish => false)
```

Comment:
I see you using both `ProgressBar.create` and `ProgressBar.new` in this PR.  Just wanted to ensure you gave that some thought.